Code/Editor/EditorFramework/Panels/AssetBrowserPanel/AssetBrowserPanel.cpp

```diff
@@ -14,7 +14,7 @@ ezQtAssetBrowserPanel::ezQtAssetBrowserPanel()
 {
   setupUi(this);
 
-  m_pStatusBar = new QStatusBar(this);
+  m_pStatusBar = new QStatusBar(nullptr);
```

Comment:
why?

Reply:
It's added manually to the dockWidgetContents later so adding it to 'this' will just unneccessarily reparent it again.